pandas/tests/series/common.py

```diff
@@ -0,0 +1,30 @@
+from pandas.util.decorators import cache_readonly
+import pandas.util.testing as tm
+import pandas as pd
+
+_ts = tm.makeTimeSeries()
+
+
+class TestData(object):
+
+    @cache_readonly
+    def ts(self):
+        ts = _ts.copy()
+        ts.name = 'ts'
+        return ts
+
+    @cache_readonly
+    def series(self):
+        series = tm.makeStringSeries()
+        series.name = 'series'
```

Comment:
not directly related, and maybe I am missing something, but these have to be strictly read only, enforced by the user. IOW. If in a single test class I modify one of these (say set an element of the Series), won't that persist?